yt/fields/derived_field.py

```diff
@@ -389,6 +389,16 @@ def _ion_to_label(self):
             "17": "XVIII",
             "18": "XIX",
             "19": "XX",
+            "20": "XXI",
+            "21": "XXII",
+            "22": "XXIII",
+            "23": "XXIV",
+            "24": "XXV",
+            "25": "XXVI",
+            "26": "XXVII",
+            "27": "XXVIII",
+            "28": "XXIX",
+            "29": "XXX",
```

Comment:
Why is this change necessary?

Reply:
If I were to guess, it's that the ionization states are used in the same place as this new field interpolator

Reply:
Yeah, that's right. Sometimes, you just need more ions.